samples/star/src/main/kotlin/com/slack/circuit/star/petlist/PetList.kt

```diff
@@ -181,10 +192,11 @@ constructor(
     val animals = animalState
     return when {
       animals == null -> PetListScreen.State.Loading
-      animals.isEmpty() -> PetListScreen.State.NoAnimals
+      animals.isEmpty() -> PetListScreen.State.NoAnimals(isRefreshing)
       else ->
         PetListScreen.State.Success(
           animals = animals.filter { shouldKeep(filters, it) },
+          isRefreshing = isRefreshing,
```

Comment:
nit: name this one too

Reply:
this one is named..?

Reply:
oh wat, github UI made that look diff :think